src/app/application.cpp

```diff
@@ -804,6 +807,30 @@ void Application::applyMemoryWorkingSetLimit()
 #endif
 }
 
+void Application::adjustThreadPriority() const
+{
+    // Workaround to raise Qt threads to be just one level higher than libtorrent threads.
+    // This is to improve responsiveness of qbt
+
+#ifdef Q_OS_WIN
```

Comment:
```suggestion
#if defined(Q_OS_WIN)
```
...to be more consistent with another branches.

Reply:
This is code obsolete now.
Also note that C++23 has added `#elifdef` and `#elifndef`. I think in the long term, C++ would prefer users using it.